util/context.go

```diff
@@ -0,0 +1,42 @@
+package util
+
+import (
+	"context"
+	"google.golang.org/grpc/metadata"
+
+	"github.com/infobloxopen/atlas-app-toolkit/auth"
+	"github.com/infobloxopen/atlas-app-toolkit/gateway"
+	"github.com/infobloxopen/atlas-app-toolkit/requestid"
+)
+
+// ForwardContext set to outgoing context request_id, auth_token and X-Forwarded-For header value
+func ForwardContext(ctx context.Context) context.Context {
```

Comment:
I think OutgoingContext would be a better name here. These are smaller helpers around metadata.NewOutgoingContext()

Reply:
Agree